### PR TITLE
Updates for jscofe and changes so that all unittests pass

### PR DESCRIPTION
**Pull request by linucks**

### CODE REVIEW

File: ample/ensembler/subcluster.py [hunk 1]
```diff
@@ -61,66 +61,50 @@ def _cluster_indices(self,thresh):
...
+
         # get mask of all elements where condition is true. We exclude 0.0 to ensure we don't get the
-        # index of the model that the row is compared with, as this needs to be the first model in the 
+        # index of the model that the row is compared with, as this needs to be the first model in the
         # ensemble. This means we would also exclude models that had an rmsd of zero to the centroid, but
         # as these are likely to be identical (and this occurrence rare), this should be ok
         condition = numpy.logical_and(self.distance_matrix <= thresh, self.distance_matrix != 0.0)
+
         # Array of sums of each row - largest number is a row where most items satisfy condition
         condition_sum =  sum(condition)
```
Comment (fsimkovic) [inline ample/ensembler/subcluster.py]: Given that you use `numpy` for all, maybe `np.sum(condition)` or `conditions.sum()` would be better/faster

Comment (linucks) [reply ample/ensembler/subcluster.py]: Nice spot! Have pushed that change.

File: ample/main.py [hunk 2]
```diff
@@ -423,7 +422,7 @@ def monitor():
 
     def process_command_line(self, args=None, contacts=True, modelling=True, mol_rep=True):
         """Process the command-line.
-        :args: optional argument that can hold the command-line arguments if we 
+        :args: optional argument that can hold the command-line arguments if we
```
Comment (fsimkovic) [inline ample/main.py:425]: Please change the docstring to numpydoc style

File: ample/util/pdb_edit.py [hunk 3]
```diff
@@ -1382,12 +1379,12 @@ def renumber_residues_gaps(pdbin, pdbout, gaps, start=1):
 def Xselect_residues(inpath=None, outpath=None, residues=None):
```
Comment (fsimkovic) [inline ample/util/pdb_edit.py]: maybe remove all the `X<something>` functions? We clearly don't use them anymore

**Review by fsimkovic — APPROVED** [review]
Looks fine to me although I would remove the redundant functions in `pdb_edit`

File: ample/util/pdb_edit.py (hunk 3 above)
Comment (linucks) [reply ample/util/pdb_edit.py]: Good idea. Done

**Pull request merged**